### PR TITLE
Add aggregated usage report fields for bot joins and issued certs

### PR DESCRIPTION
**Pull request by timothyb89**
This adds two new fields to reports from the aggregating reporter:
* `certificates_issued` counts the number of `cert.create` events related to the given user
* `bot_joins` counts the number of `bot.join` events for the user

See also: https://github.com/gravitational/teleport.e/issues/3200

### CODE REVIEW

**Comment by timothyb89** [comment]
On hold for the moment. We have some prerequisite changes for Cloud to make sure new fields added here are counted correctly.

**Comment by timothyb89** [comment]
TIA has been deprecated in Cloud so this is unblocked.

**Review by mdwn — APPROVED** [review]

**Review by strideynet — APPROVED** [review]

**Review by espadolini — APPROVED** [review]

**Comment by espadolini** [comment]
Needs gravitational/cloud#7396 before merging.

**Pull request merged**
